contrib/systemd/atomic-openshift-master.service

```diff
@@ -1,10 +1,10 @@
 [Unit]
 Description=Atomic OpenShift Master
 Documentation=https://github.com/openshift/origin
-After=network.target
+After=network-online.target
 After=etcd.service
-Before=atomic-openshift-node.service
-Requires=network.target
+Requires=network-online.target
+Wants=etcd.service
```

Comment:
Can we get by without Wants=etcd.service ? When we first put these together we didn't want to start etcd if it weren't enabled.